src/App.jsx

```diff
@@ -22,7 +25,10 @@ function App() {
         <Route path="home" element={<Home />} />
         <Route path="symptome" element={<Symptome />} />
         <Route path="medikamente" element={<Medikamente />} />
+        <Route path="/medikamente" element={<Medikamente />} />
         <Route path="termine" element={<Termine />} />
+        <Route path="/neuertermin" element={<NeuerTermin />} />
+        <Route path="/neuesmedikament" element={<NeuesMedikament />} />
```

Comment:
Could all routes be absolute (`/medikamente` etc.)?

Reply:
No idea why it didn't work in the first place, now it does. Added "/" to all routes!